extension/options.js

```diff
@@ -1,18 +1,19 @@
 // Default values
 const options = {
-	previewCount: true
+	previewCount: true,
+	compactUI: true
 };
 
 chrome.storage.sync.get({options}, ({options}) => {
-	const field = document.querySelector('[name=previewCount]');
-	if (options.previewCount) {
-		field.checked = true;
-	}
-	field.addEventListener('change', () => {
-		chrome.storage.sync.set({
-			options: {
-				previewCount: field.checked
-			}
+	Object.keys(options).forEach((option, value) => {
```

Comment:
`value` here is actually just the index, which is 0, 1, 2...

The value needs to be `options[option]`

Better written as:

```js
for (const key of Object.keys(options)) {
	const value = options[key];
```

If we get different types of settings in the future it might be a good idea to just use https://github.com/bfred-it/webext-options-sync

Reply:
Why not 
```js 
for (const [option, value] of Object.entries(options))
```
or 
```js
Object.entries(options).forEach(([option, value]) => {...})
```
?

Reply:
True, forgot Object.entries

`for...of` is faster